From 3a13686ef6cee5458553b60c13aabb3c93294051 Mon Sep 17 00:00:00 2001
From: maxwen <max.weninger@gmail.com>
Date: Sat, 7 Dec 2013 17:33:32 +0100
Subject: [PATCH 05/18] base: added national roaming info for Play in Poland

Cherry-picked from OmniROM:
https://gerrit.omnirom.org/#/c/3382/

Users of Play in Poland (MCC 260 MNC 06) can roam on
Plus, T-Mobile, Orange (MNCs 01/02/03)

Change-Id: I8ea62b0ca40bac9ced11802ee8ef583d6392a199
---
 core/res/res/values-mcc260-mnc06/config.xml | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)
 create mode 100644 core/res/res/values-mcc260-mnc06/config.xml

diff --git a/core/res/res/values-mcc260-mnc06/config.xml b/core/res/res/values-mcc260-mnc06/config.xml
new file mode 100644
index 0000000..252565f
--- /dev/null
+++ b/core/res/res/values-mcc260-mnc06/config.xml
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+** Copyright 2013, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <!-- Don't use roaming icon for considered operators -->
+    <string-array translatable="false" name="config_operatorConsideredNonRoaming">
+        <item>26001</item>
+        <item>26002</item>
+        <item>26003</item>
+    </string-array>
+</resources>
-- 
2.7.4

